src/component/movie-grid/MovieGrid.ts

```diff
@@ -0,0 +1,43 @@
+import MovieItem from '../movie-item/MovieItem';
+import { ERROR_MESSAGE } from '../../constants/errorMessage';
+import ErrorMessage from '../error-message/ErrorMessage';
+
+interface MovieGridProps {
+  movieItems: MovieData[];
+}
+
+class MovieGrid {
+  #container;
+  #movieItems;
+
+  constructor({ movieItems = [] }: MovieGridProps) {
+    this.#container = document.createElement('main');
+    this.#movieItems = movieItems;
+    this.render();
+  }
+
+  render() {
+    if (this.#movieItems.length !== 0) {
+      this.#container.innerHTML = `
+      <ul class="thumbnail-list">
+      ${this.#movieItemElements()};
+      </ul>`;
```

Comment:
소소한 부분이지만 indent를 신경써주시면 코드 보기가 좋을거 같아요
```suggestion
      this.#container.innerHTML = `
        <ul class="thumbnail-list">
          ${this.#movieItemElements()};
        </ul>`;
```